src/components/common/Detail.vue

```diff
@@ -33,27 +36,36 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue, Watch } from 'vue-property-decorator'
-import { DetailComment as DetailCommentShared } from '@bcrs-shared-components/detail-comment'
+import { Component, Mixins, Prop, Watch } from 'vue-property-decorator'
+import { DetailComment as DetailCommentShared } from '@bcrs-shared-components/detail-comment/'
 import { Action, Getter } from 'vuex-class'
 import { ActionBindingIF } from '@/interfaces/'
+import { DateMixin } from '@/mixins'
 
 @Component({
   components: { DetailCommentShared }
 })
-export default class Detail extends Vue {
+export default class Detail extends Mixins(DateMixin) {
+  /** Prop to provide section number. */
+  @Prop({ default: '' }) readonly sectionNumber: string
+
   @Getter getDetailComment!: string
-  @Getter getDefaultCorrectionDetailComment!: string
+  @Getter getOriginalFilingDateTime!: string
 
   @Action setDetailComment: ActionBindingIF
   @Action setDetailValidity: ActionBindingIF
 
   private comment: string = null
-  private textAreaStyle = 'filled'
+
+  get defaultCorrectionDetailComment (): string {
+    const date = this.apiToDate(this.getOriginalFilingDateTime)
+    const yyyyMmDd = this.dateToYyyyMmDd(date)
+    return `Correction for Incorporation Application filed on ${yyyyMmDd}.`
+  }
```

Comment:
I had to move this here (instead of store getters) since a date mixin was needed.

(This is a bugfix, where the original filing date was displayed as UTC instead of Pacific.)